x/mock/simulation/util.go

```diff
@@ -0,0 +1,31 @@
+package simulation
+
+import (
+	"math/rand"
+)
+
+// shamelessly copied from https://stackoverflow.com/questions/22892120/how-to-generate-a-random-string-of-a-fixed-length-in-golang#31832326
+
+const letterBytes = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
```

Comment:
Lets move this to `tendermint/libs/common/random.go`. For now can you add a `TODO` comment so we remember to change this whenever that update lands there?